src/internal/event-bridge.ts

```diff
@@ -0,0 +1,139 @@
+import { JSONObject } from 'k6'
+import http, { RefinedResponse, ResponseType } from 'k6/http'
+
+import { AWSClient } from './client'
+import { AWSConfig } from './config'
+import { AWSError } from './error'
+import { InvalidSignatureError, SignatureV4 } from './signature'
+import { AMZ_TARGET_HEADER } from './constants'
+import { HTTPHeaders, HTTPMethod } from './http'
+
+type PutEventEntry = {
+    Detail: string
```

Comment:
The [documentation](https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEventsRequestEntry.html) states "A valid JSON object. There is no other schema imposed. The JSON object may contain fields and nested sub-objects.".

Later in the code, you `JSON.stringify` this field lets me think you expect an actual `Object` to be passed in. 

Judging by how this will be used (disclaimer: I haven't used event bridge yet), I believe the fact that it is sent as a string to the AWS API is an implementation detail and that exposing it to the user as an `Object` might lead to a better developer experience.

I'm not certain we should change it, but for reference, I believe we could take this opportunity to harness the typescript type system power and define a type for it:

```typescript
type JsonValue = string | number | boolean | null | JsonArray | JsonObject;
type JsonArray = JsonValue[];
interface JsonObject { [key: string]: JsonValue; }

interface PutEventEntry {
    Detail: JsonObject;  // or JsonValue if you want to be more permissive
    DetailType: string;
    EventBusName: string;
    Resources: string[];
    Source: string;
}
```

This introduces some abstraction for an allegedly small benefit, and once again I'm not a user, so I don't know if it would feel better that way.

What do you think?

Reply:
Good spot! It was a design decision to have `Detail` as a JSON object type (see the test), but I had specified the wrong type of `string`. I'll update as per your suggestion here 👍

Reply:
Great, thanks you 🙏🏻